ops/src/test/scala/test/ammonite/ops/PathTests.scala

```diff
@@ -340,6 +339,42 @@ object PathTests extends TestSuite{
           }
         }
       }
+      'symlinks{
+        'nestedSymlinks{
+          if(Unix()) {
+            val names = Seq('test123, 'test124, 'test125, 'test126)
+            names.foreach(p => rm ! pwd / p)
+            mkdir ! pwd / 'test123
+            ln.s(pwd / 'test123, pwd / 'test124)
+            ln.s(pwd / 'test124, pwd / 'test125)
+            ln.s(pwd / 'test125, pwd / 'test126)
+
+            assert((pwd / 'test126).tryFollowLinks.get == pwd / 'test123)
+
+            names.foreach(p => rm ! pwd / p)
+            names.foreach(p => assert(!exists(pwd / p)))
+          }
+
+        }
+        'danglingSymlink{
+          if(Unix()) {
+            val names = Seq('test123, 'test124, 'test125, 'test126)
+            names.foreach(p => rm ! pwd / p)
+            mkdir ! pwd / 'test123
+            ln.s(pwd / 'test123, pwd / 'test124)
+            ln.s(pwd / 'test124, pwd / 'test125)
+            ln.s(pwd / 'test125, pwd / 'test126)
+
+            rm ! pwd / 'test123
+
+            assert( (pwd / 'test126).tryFollowLinks.isEmpty)
+
+            names.foreach(p => rm ! pwd / p)
+            names.foreach(p => assert(!exists(pwd / p)))
+          }
+
+        }
+      }
```

Comment:
Could we add some simple tests for `mkdir`, `write`, `read`, `stat`/`stat.full`, `cp`, and `ls` inside a sym-linked directory? I'm 99% they will work, but we should verify